jupyter/README.md

```diff
@@ -0,0 +1,67 @@
+# Spin up Jupyter Notebooks with Trace DSL
+
+Jupyter notebooks provide a simple user interface for experimenting with data. There are two main use cases that we want to accomplish with the notebooks:
+
+- an interface for data scientists to experiment with tracing data
+- on-demand incident investigation
+
+The second point is the reason why we have developed Trace DSL based on graph query language Gremlin to simplify filtering and feature extraction of tracing data. Because such analysis might not be feasible in the user interface where thousands of traces or set of traces with hundreds or thousands of spans are to be analysed. Hence a Jaeger user would be able to spin up a Jupyter notebook on demand with the Trace DSL and write a query and analysis.
+
+## Jupyter notebook example with Jaeger
+
+Now, let's see how we can deploy Jupyter notebook with Jaeger trace DSL and write a simple query against the Jaeger server.
+
+For this, first we need to deploy Jaeger UI, HotROD - a demo application and Jupyter notebook with Trace DSL.
+
+1. For Jaeger UI: 
+
+``` bash
+docker run --rm -it -p 16686:16686 --name=jaeger jaegertracing/all-in-one:1.20
+```
+This will open up the Jager UI at [http://localhost:16686](http://localhost:16686/)
+
+2. For HotROD demo application, run
+``` bash 
+docker run --rm -it -p 8080:8080 --link=jaeger -e JAEGER_ENDPOINT=http://jaeger:14268/api/traces jaegertracing/example-hotrod:1.17
+```
+This will start the application at [http://localhost:8080](http://localhost:8080/)
+
+3. Lastly, for spinning up the notebook that are hosted in this [repository](https://github.com/jaegertracing/jaeger-analytics-java)
+
+
+``` bash
+docker run --rm -it -p 8888:8888 -p 4041:4040 -p 9001:9001 --link=jaeger -e JUPYTER_ENABLE_LAB=yes quay.io/jaegertracing/jaeger-analytics-java:latest
```

Comment:
`-p 4041:4040` -- is it a typo, or should one really be different than the other?